packages/js/src/dashboard/widgets/site-kit-setup-widget.js

```diff
@@ -134,43 +132,44 @@ export const SiteKitSetupWidget = ( { dataProvider, remoteDataProvider } ) => {
 				</DropdownMenu.ButtonItem>
 			</DropdownMenu.List>
 		</DropdownMenu>
-		<div className="yst-flex yst-justify-center yst-mb-6 yst-mt-4"><YoastConnectSiteKit width="252" height="60" /></div>
+		<div className="yst-flex yst-justify-center yst-mb-6 yst-mt-4">
+			{ isSiteKitConnectionCompleted  ? <YoastConnectSiteKitSuccess width="252" height="60" />
+				: <YoastConnectSiteKit width="252" height="60" /> }
```

Comment:
It would be more readable to me if the multiline would be split per component and the ending is on its own line, like so:

```suggestion
			{ isSiteKitConnectionCompleted
				? <YoastConnectSiteKitSuccess width="252" height="60" />
				: <YoastConnectSiteKit width="252" height="60" />
			}
```